client/extensions/woocommerce/app/order/order-details/table.js

```diff
@@ -114,6 +116,20 @@ class OrderDetailsTable extends Component {
 		this.props.onChange( { line_items: { [ index ]: newItem } } );
 	};
 
+	onShippingChange = event => {
+		const { order } = this.props;
+		const shippingLine = order.shipping_lines[ 0 ];
```

Comment:
Is there ever any chance an order could have more than one shipping line here?

Reply:
I think you can with extensions, so it's definitely something to keep in mind as we expand our support, but for now I thought this would be OK.